test/evergreen.yml

```diff
@@ -1576,8 +1582,15 @@ tasks:
             set -o errexit
             set -o verbose
 
-            pip install scons==3.1.1
-            scons-3.1.1.bat ${smp_command|} "CFLAGS=/Gv /wd4090 /wd4996 /we4047 /we4024 /TC /we4100 /we4133" wiredtiger.dll libwiredtiger.lib
+            export "PATH=/cygdrive/c/Python39:/cygdrive/c/Python39/Scripts:$PATH"
+            python -m pip install scons==3.1.1
+            python --version
+            echo python --version > foo.bat
+            chmod +x foo.bat
+            echo running foo.bat
+            ./foo.bat
+            echo running scons-3.1.1.bat ${smp_command|} "CFLAGS=/Gv /wd4090 /wd4996 /we4047 /we4024 /TC /we4100 /we4133" "LIBPATH=c:\\python\\Python37\\libs" wiredtiger.dll libwiredtiger.lib
```

Comment:
Evergreen will print the command executed into the task log, so this line might not be necessary, unless you'd like to keep it for running locally?

Reply:
Fixed.